<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clock</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="Time_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>190</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Time_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>190</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Time_3">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>190</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="time_edit_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="time_edit_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>180</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="time_edit_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
